pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ThresholdShedder.java

```diff
@@ -101,6 +102,14 @@
                     TimeAverageMessageData shortTermData = bundleData.getShortTermData();
                     double throughput = shortTermData.getMsgThroughputIn() + shortTermData.getMsgThroughputOut();
                     return Pair.of(bundle, throughput);
+                }).filter(e -> {
+                    // We cannot unload the bundle that contains the Heartbeat of the broker
+                    final String namespaceName = LoadManagerShared.getNamespaceNameFromBundleName(e.getKey());
+                    boolean isHeartbeat  = NamespaceService.isHeartBeatNamespace(namespaceName);
+                    if (isHeartbeat) {
+                        log.info("Skipping Heartbeat bundle {}", e.getKey());
```

Comment:
debug level ? it will come up every time and it's a normal behaviour

Reply:
I agree. changed